source/grid/grid_generator.cc

```diff
@@ -3829,7 +3829,238 @@ namespace GridGenerator
           face->set_boundary_id(3);
   }
 
+  template <>
+  void
+  custom_channel_with_cylinder(
+    [[maybe_unused]] Triangulation<1>  &tria,
+    [[maybe_unused]] const unsigned int half_height,
+    [[maybe_unused]] const unsigned int length_pre,
+    [[maybe_unused]] const unsigned int length_post,
+    [[maybe_unused]] const double       depth,
+    [[maybe_unused]] unsigned int       depth_division,
+    [[maybe_unused]] const double       shell_region_radius,
+    [[maybe_unused]] const unsigned int n_shells,
+    [[maybe_unused]] const double       skewness,
+    [[maybe_unused]] const bool         colorize)
```

Comment:
```suggestion
    Triangulation<1>  &,
    const unsigned int,
    [[maybe_unused]] const unsigned int length_pre,
    [[maybe_unused]] const unsigned int length_post,
    [[maybe_unused]] const double       depth,
    [[maybe_unused]] unsigned int       depth_division,
    [[maybe_unused]] const double       shell_region_radius,
    [[maybe_unused]] const unsigned int n_shells,
    [[maybe_unused]] const double       skewness,
    [[maybe_unused]] const bool         colorize)
```
I'd prefer to do it this way since (pre C++17) this is how we handled unused arguments in overloads we don't plan on calling.